server/src/workflows/2020-12-01/WorkflowService.ts

```diff
@@ -37,6 +37,8 @@ export class WorkflowService20201201 {
     switch (runtimeStack) {
       case RuntimeStacks.Dotnet:
         return this.readWorkflowFile('function-app-configs/dotnetcore-linux.config.yml');
+      case RuntimeStacks.DotnetIsolated:
+        return this.readWorkflowFile('function-app-configs/dotnetcore-linux.config.yml');

```

Comment:
I wanted to double-check that these two cases should return the same value. If so:

```suggestion
      case RuntimeStacks.Dotnet: // falls through
      case RuntimeStacks.DotnetIsolated:
        return this.readWorkflowFile('function-app-configs/dotnetcore-linux.config.yml');
```